packages/layout/header.jade

```diff
@@ -1,2 +1,11 @@
-template(name='header')
-  p HEADER
\ No newline at end of file
+template(name="header")
+  header
+    nav.navbar
+      .fluid-container
+        .container
```

Comment:
I established that structure when I was learning bootstrap. ```.fluid-container``` adds padding on the left and right which is necessary at the smallest breakpoint. I'll see how we can remove the .fluid-container and keep the padding.